src/code42cli/cmds/securitydata.py

```diff
@@ -211,31 +216,21 @@ def search(
         include_all, SEARCH_DEFAULT_HEADER, format
     )
     format_func = get_file_events_output_format_func(format)
-    cursor = (
-        _get_file_event_cursor_store(state.profile.name) if use_checkpoint else None
-    )
-    handlers = ext.create_handlers(
-        state.sdk,
-        FileEventExtractor,
-        cursor,
+    print_events_decorator = print_events(format_func, output_header)
+
+    handlers = _extract_events(
```

Comment:
Since this is only used for `search`, we don't need a separate method like this... Mostly, I'd like to get rid of `output_processor`

Reply:
Changed. Removed.